core/domain/user_jobs_one_off.py

```diff
@@ -429,8 +426,6 @@ def map(user_subscriptions_model):
             del user_subscriptions_model._properties['activity_ids']  # pylint: disable=protected-access
             if 'activity_ids' in user_subscriptions_model._values:  # pylint: disable=protected-access
                 del user_subscriptions_model._values['activity_ids']  # pylint: disable=protected-access
-            user_subscriptions_model.update_timestamps(
-                update_last_updated_time=False)
```

Comment:
Are there any repercussions to always updating the timestamp now? This seems to be a behavioral change from before for models that aren't leveraging the new human-maintained model.

Reply:
No, we decided that we want to implement the ability to specify which kind of update was done only for models that actually use the `last_updated` or more precisely `last_updated_by_human` field.

Reply:
Just to confirm: I'm interpreting your 'no' to be an answer to the question about whether there are any repercussions--is that correct?